lib/storage/datauri.js

```diff
@@ -27,7 +28,33 @@ async function download(source, file) {
         throw new GenericError(err.message, `${actionName()}_download`);
     }
 }
+/**
+ * Generate Presign Url taking local file path as input
+ * Creates a unqiue name for the file, upload the file to 
+ * storage from the path provided and generates a PreSigned Url
+ * @param {String} Path to local file system, where file is downloaded
+ * @returns {String} PreSigned Url of upload the source file
+ */
+async function getPreSignedUrl(source) {
+    /**
+     * if is local file, upload to cloud storage and return signed URL
+     */
+    const temporaryCloudStorage = new TemporaryCloudStorage();
+    // Create temp file name,
+    const path = temporaryCloudStorage.createUniqueName();
+    // Upload
+    console.log(`Source file location is ${source}, and unique path for the file is ${path}`);
+    await temporaryCloudStorage.upload(source, path);
+    console.log(`${source} file is uploaded to cloud at path ${path}`);
```

Comment:
I would not log the cloud path, in case someone could consult the logs and try to grab the file